backend_addon/{{ cookiecutter.__folder_name }}/Makefile

```
@@ -92,10 +91,34 @@ console: instance/etc/zope.ini ## Start a console into a Plone instance
 create-site: instance/etc/zope.ini ## Create a new site from scratch
 	PYTHONWARNINGS=ignore $(BIN_FOLDER)/zconsole run instance/etc/zope.conf ./scripts/create_site.py
 
-.PHONY: check
-check: $(BIN_FOLDER)/tox ## Check and fix code base according to Plone standards
+# QA
+.PHONY: lint-zpretty
+lint-zpretty: ## Check and fix zcml and xml files
+	@find src -name '*.xml'|xargs uvx zpretty@latest -x --check
+	@find src -name '*.zcml'|xargs uvx zpretty@latest -z --check
+
+.PHONY: lint
+lint: $(BIN_FOLDER)/uv ## Check and fix code base according to Plone standards
+	@echo "$(GREEN)==> Lint codebase$(RESET)"
+	@uvx ruff@latest check --fix --config $(BACKEND_FOLDER)/pyproject.toml
+	@uvx pyroma@latest -d .
+	@uvx check-python-versions@latest .
+	@make lint-zpretty
+
+.PHONY: format-zpretty
+format-zpretty: ## Check and fix zcml and xml files
+	@find src -name '*.xml'|xargs uvx zpretty@latest -x -i
+	@find src -name '*.zcml'|xargs uvx zpretty@latest -z -i
```

This should be equivalent
```suggestion
	@uvx zpretty@latest -i --include="\.(xml|zcml)$" src 
```